src/component/toolbox/feature/DataView.ts

```diff
@@ -534,4 +540,14 @@ echarts.registerAction({
     }, payload.newOption));
 });
 
+echarts.registerAction(
+    { type: 'openDataView', event: 'dataViewOpend' },
```

Comment:
Event name can be `dataViewShowed` and `dataViewHidden`.

Reply:
Should it be `dataViewShown` instead of `dataViewShowed`?

Reply:
@MainHou And here we can use provided `noop` from zrender util.
```ts
import { noop } from 'zrender/src/core/util';
```

Reply:
@Ovilia @MainHou I noticed the registered action doesn't respond to the corresponding command from the developer. That is, this PR is only for the event but not for action, which causes the developer can't open or close the data view programmatically. I think it makes sense to support this feature.

```js
// show the data view programmatically
myChart.dispatchAction({
  type: 'showDataView'
})

// hide the data view programmatically
myChart.dispatchAction({
  type: 'hideDataView'
})
```